### PR TITLE
VMware: new module : vmware_about_facts

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY
Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
 - New Module Pull Request

##### COMPONENT NAME
lib/ansible/modules/cloud/vmware/vmware_about_facts.py
test/integration/targets/vmware_about_facts/aliases
test/integration/targets/vmware_about_facts/tasks/main.yml

##### ANSIBLE VERSION

```
2.7-devel
```

### CODE REVIEW

**Comment by Akasurde** [comment]
@dericcrago @pdellaert @tchernomax Could you please review this ?

File: lib/ansible/modules/cloud/vmware/vmware_about.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+'''
+
+RETURN = r'''
+about_facts:
+    description:
+    - dict about VMware server
+    returned: success
+    type: string
+    sample:
+
```
Comment (pdellaert) [inline lib/ansible/modules/cloud/vmware/vmware_about.py]: Can you add an example?

**Comment by jctanner** [comment]
I think the naming is less than obvious. Something more like "vmware_service_facts" might make more sense.

File: lib/ansible/modules/cloud/vmware/vmware_about.py (hunk 1 above)
Comment (Akasurde) [reply lib/ansible/modules/cloud/vmware/vmware_about.py]: ah! missed it.

**Comment by pdellaert** [comment]
@jctanner , what about `vmware_about_system`. Problem i have with `service` is that there are 'services' like ntp and others you can gather facts from as well in vCenter or an ESXi host... 

or it could be `vmware_about_info`, to keep it in line with the API object: https://code.vmware.com/apis/358/vsphere#/doc/vim.AboutInfo.html (but i find that confusing if you don't know the API well)

**Comment by Akasurde** [comment]
I am OK with `vmware_about_system`. Also, `vmware_about_facts` is good option.

**Comment by pdellaert** [comment]
either are good to me :)

**Comment by Akasurde** [comment]
@pdellaert Can you take another look into this ?

**Review by pdellaert — APPROVED** [review]
LGTM :)

**Pull request merged**